report/renderer/i18n.js

```diff
@@ -32,36 +32,42 @@ export class I18n {
 
   /**
    * @param {number} number
-   * @param {number} granularity
+   * @param {number|undefined} granularity
    * @param {Intl.NumberFormatOptions} opts
    * @return {string}
    */
   _formatNumberWithGranularity(number, granularity, opts = {}) {
-    opts = {...opts};
-    const log10 = -Math.log10(granularity);
-    if (!Number.isFinite(log10) || (granularity > 1 && Math.floor(log10) !== log10)) {
-      throw new Error(`granularity of ${granularity} is invalid`);
-    }
+    if (granularity !== undefined) {
```

Comment:
it does feel like we're working too hard to maintain `granularity` which was kind of a haphazard choice long ago, but it's unfortunately in the LHR, so...:(